src/null-safety/index.md

```diff
@@ -314,9 +60,23 @@ To find the most recent releases, see the
 Dart SDK archive, or the **Beta channel** section of the
 [Flutter SDK archive.][flutter-sdks]
 
+### Creating a null-safe package or app
 
-Set the [SDK constraints](/tools/pub/pubspec#sdk-constraints)
```

Comment:
I feel the migration use case is getting a bit lost in between the `## Enabling null safety` section and the `### Behind the scenes` below.

I think I'd prefer something like:

```
## Enabling null safety

Current intro...

### Migrating existing packages or apps

Migrating existing code require a number of steps; see the [migration guide][].

### Creating a null-safety package or app
...
```